mac

```diff
@@ -113,6 +113,14 @@ gem_install_or_update() {
   fi
 }
 
+find_latest_ruby() {
+  local ruby_download_site="https://www.ruby-lang.org/en/downloads/"
+
+  fancy_echo "Find latest stable Ruby version ..."
+  curl -s $ruby_download_site | grep "Current stable:" -A 1 | grep "Ruby" |\
+    sed 's|</b>|-|g' | sed 's|<[^>]*>||g' | sed 's/Ruby //g'
```

Comment:
thanks @mike-burns... any thoughts on going this route verse what @iwz suggested? Since this is parsing a webpage it seems like it might have more of a potential to break in the future.